components/micro-gateway-core/src/main/ballerina/src/gateway/handlers/jwt_auth_handler.bal

```diff
@@ -117,12 +114,16 @@ public type JWTAuthHandler object {
 
 # Check whether backendJwt claim is in the payload and set the header if avaialable.
 #
+# + jwtValidatorConfig - jwtValidatorConfig to access the jwtCache object
 # + credential - Credential
 # + req - The `Request` instance.
 # + issuer - The jwt issuer who issued the token and comes in the iss claim.
```

Comment:
Remove the issuer ballerina doc parameter. This gives a warning when building the mgw project.

Reply:
that PR is reverted.